src/migrations/20191231150619-create-trip.js

```diff
@@ -8,8 +8,8 @@ export const up = (queryInterface, Sequelize) => queryInterface.createTable('Tri
   tripType: {
     type: Sequelize.STRING
   },
-  tripId: {
-    type: Sequelize.BIGINT
+  requestId: {
```

Comment:
Create a new migration instead of editing the existing one. Also, ensure you drop the `tripId` column in the new migration

Reply:
Also, add a relation reference to the migration